README.md

```diff
@@ -1,5 +1,278 @@
+                            faker.getWords(2),
+                            faker.getWords(1)))
+val row3 = textRow(textCell(faker.getWords(5),
+                            faker.getWords(2),
+                            faker.getWords(1)),
+                   textCell(faker.getWords(2),
+                            faker.getWords(2)),
+                   textCell(faker.getWords(3)))
+```
+
+Then, these values can be used in the following ways:
+
+##### Option 1
+
+```kotlin
+textTable(row1, row2, row3) {
+    withHeader(header)
+    withHeaderDivider(divider)
+}.println()
+```
+
+##### Option 2
+
+```kotlin
+textTable(listOf(row1, row2, row3)) {
+    withHeader(header)
+    withHeaderDivider(divider)
+}.println()
+```
+
+##### Option 3
+
+```kotlin
+textTable {
+    withBody(row1, row2, row3)
+    withHeader(header)
+    withHeaderDivider(divider)
+}.println()
+```
+
+##### Option 4
+
+```kotlin
+textTable {
+    addBody(row1)
+    addBody(row2)
+    addBody(row3)
+    withHeader(header)
+    withHeaderDivider(divider)
+}.println()
+```
+
+#### Java API
+
+#### Sample Output
+
+```
+In                                        |Mauris             |Omittam                     
+------------------------------------------|-------------------|----------------------------
+percipit bibendum purus augue cum         |dolores sea        |graecis voluptaria salutatus
+iisque phasellus                          |                   |inciderint vix              
+                                          |                   |mea                         
+antiopam porta omittantur mazim erroribus |option himenaeos   |consul nobis contentiones   
+                                          |aptent oporteat    |convenire dicam             
+                                          |                   |decore                      
+mi referrentur conclusionemque hac dapibus|iisque deterruisset|verear doming commune       
+duo tractatos                             |simul alienum      |                            
+autem                                     |                   |                            
+```
+
+### Blocked Text
+
+#### Kotlin API
+
+#### Java API
+
+## Issues
+
+[GitHub: Issues](https://github.com/NexusSandbox/inevitable/issues)
+- Please include as much of the following as is relevant:
+  - Title -
+    - A concise description of the issue
+  - Summary - 
+    - A detailed description of the issue
+    - An accurate example of the workflow that triggers the issue
+      - Include code and/or screenshots as necessary
+    - The applicable versions of the code and environment you are using
+      - Version of `inevitable` consumed?
+      - Version of the JVM/Kotlin?
+      - Version of MacOS or Windows?
+
+## Contributions
+
+I welcome any contributions and suggestions in hopes that this library become as useful and helpful as possible. However, I would request that you abide by the following for any changes:
```

Comment:
nit: spacing

```
However, I would request ...:

* item1
* item2
```

Reply:
Are you suggesting that "However ..." part should be on its own line?

Reply:
No, the line should be between the : and the bullet items

Reply:
Oh, alrighty! Fixed